R/volcano_protti.R

```diff
@@ -0,0 +1,115 @@
+#' Volcano plot
+#'
+#' Plots a volcano plot for the given input
+#'
+#' @param data Dataframe containing at least sample names, peptide or precursor identifiers and missed cleavage counts for each peptide or precursor.
+#' @param grouping Column in the data dataframe containing either precursor or peptide identifiers.
+#' @param foldchange Column in the dataframe containing the fold changes (ideally log2 transformed) between two conditions
+#' @param p_value Column containing the p-value for the corresponding fold changes. P-value is ideally adjusted using e.g. Benjamini-Hochberg correction.
```

Comment:
would not change the column p_value but rather significance since it would also be a q_value derived from BH.